Podcast/RecommendedSeriesCollectionViewCell.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  RecommendedSeriesCollectionViewCell.swift
+//  Podcast
+//
+//  Created by Kevin Greer on 2/19/17.
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+
+class RecommendedSeriesCollectionViewCell: UICollectionViewCell {
+    
+    let kImageTitlePadding: CGFloat = 8
```

Comment:
remove the k in front of the constant names, I think you picked this up from really old objective c standards